### PR TITLE
LPS-78096

### PR DESCRIPTION
**Pull request by petershin**

### CODE REVIEW

**Comment by petershin** [comment]
ci:test

File: .m2-tmp/com/liferay/com.liferay.gradle.plugins.poshi.runner/2.0.1/com.liferay.gradle.plugins.poshi.runner-2.0.1.pom [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
```
Comment (Ithildir) [inline .m2-tmp/com/liferay/com.liferay.gradle.plugins.poshi.runner/2.0.1/com.liferay.gradle.plugins.poshi.runner-2.0.1.pom]: I think you should run `gradlew baseline` on `gradle-plugins-poshi-runner`.

File: modules/sdk/gradle-plugins-defaults/CHANGELOG.markdown [hunk 2]
```diff
@@ -2878,6 +2878,12 @@ applies the `com.liferay.app.jsdoc` plugin.
 ### Changed
 - [LPS-78033]: Update the [Liferay Gradle Plugins] dependency to version 3.7.4.
 
+## 5.3.5 - 2018-02-16
```
Comment (Ithildir) [inline modules/sdk/gradle-plugins-defaults/CHANGELOG.markdown]: You should create a changelog for `gradle-plugins-poshi-runner`.

File: modules/sdk/gradle-plugins-defaults/src/main/java/com/liferay/gradle/plugins/defaults/PoshiRunnerResourcesDefaultsPlugin.java [hunk 3]
```diff
@@ -63,6 +64,12 @@ private void _applyConfigScripts(Project project) {
 	private void _configureTaskUploadPoshiRunnerResources(
 		final Project project) {
 
+		PoshiRunnerResourcesExtension poshiRunnerResourcesExtension =
```
Comment (Ithildir) [inline modules/sdk/gradle-plugins-defaults/src/main/java/com/liferay/gradle/plugins/defaults/PoshiRunnerResourcesDefaultsPlugin.java]: Move this into a `_configurePoshiRunnerResources(Project)` method.

File: modules/sdk/gradle-plugins-poshi-runner/src/main/java/com/liferay/gradle/plugins/poshi/runner/PoshiRunnerResourcesPlugin.java [hunk 4]
```diff
@@ -103,9 +122,10 @@ private void _addArtifactsPoshiRunnerResources(
 		for (Map.Entry<Object, Object> entry : baseNameDirs.entrySet()) {
 			String baseName = GradleUtil.toString(entry.getKey());
 			File dir = GradleUtil.toFile(project, entry.getValue());
+			String rootDirName = poshiRunnerResourcesExtension.getRootDirName();
```
Comment (Ithildir) [inline modules/sdk/gradle-plugins-poshi-runner/src/main/java/com/liferay/gradle/plugins/poshi/runner/PoshiRunnerResourcesPlugin.java]: Please move this outside of the loop.

**Comment by Ithildir** [comment]
ci:stop

**Comment by petershin** [comment]
Applied the changes: https://github.com/Ithildir/liferay-portal/pull/891

I'll send the changelog for poshi-runner in a separate pull

**Comment by Ithildir** [comment]
No problem, I can take care of it 😊

**Pull request closed without merging**